tools/ionizer/options.c

```diff
@@ -10,8 +10,8 @@
 #include <stdio.h>
 #ifdef WIN
 #include <io.h>
-#endif
 #include <malloc.h>
+#endif
```

Comment:
My understanding is that `malloc.h` is an archaic header for Linux systems, and is probably not on MSVC.  Perhaps we should eliminate this include all together or prefer `stdlib.h`